distribution/lib/Standard/AWS/0.0.0-dev/package.yaml

```diff
@@ -8,3 +8,6 @@ authors:
 maintainers:
   - name: Enso Team
     email: contact@enso.org
+requires:
+  - Standard.Base
+  - Standard.Database
```

Comment:
With 57e64e0d69fc332a93664593d765c35ba9ba10b5 I am getting the following when running `test/AWS_Tests`
```
Execution finished with an error: java.lang.module.ResolutionException: Modules aws.java.sdk.sts and aws.java.sdk.redshift export package com.amazonaws.auth.policy.actions to module software.amazon.awssdk.metrics
        at <java> java.base/java.lang.module.Resolver.resolveFail(Resolver.java:900)
        at <java> java.base/java.lang.module.Resolver.failTwoSuppliers(Resolver.java:814)
        at <java> java.base/java.lang.module.Resolver.checkExportSuppliers(Resolver.java:735)
        at <java> java.base/java.lang.module.Resolver.finish(Resolver.java:380)
        at <java> java.base/java.lang.module.Configuration.<init>(Configuration.java:139)
        at <java> java.base/java.lang.module.Configuration.resolveAndBind(Configuration.java:493)
        at <java> org.enso.runtime/org.enso.interpreter.runtime.EnsoClassPath.create(EnsoClassPath.java:62)
```
We are not the only ones seeing this problem: https://github.com/aws/aws-sdk-java/issues/1658#issuecomment-430932125

Reply:
Solved by creating our own [aws-wrapper](https://github.com/enso-org/enso/pull/10714/files#diff-5634c415cd8c8504fdb973a3ed092300b43c4b8fc1e184f7249eb29a55511f91R1536). Thanks @Akirathan for your help!